src/smc-webapp/app-framework/TypedMap.ts

```diff
@@ -119,7 +119,7 @@ interface TypedMapFactory<TProps extends Record<string, any>> {
   new (values: TProps): TypedMap<TProps>;
 }
 
-export function typedMap<TProps extends object>(
+export function TypedMap<TProps extends object>(
```

Comment:
Follow immutable.js's convention that the type is also the function to create one.